game/scripts/npc/abilities/lone_druid_spirit_bear.txt

```diff
@@ -57,7 +57,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "backlash_damage"                                 "20 30 40 50 60 70" //OAA
+        "backlash_damage"                                 "20.0"
```

Comment:
why?

Reply:
It said field float so I added a .0

Reply:
I meant why buff Lone Druid?

Reply:
good change, backlash dmg is dumb anyways @DarkoniusXNG